force-app/main/default/classes/AdyenClient.cls

```diff
@@ -29,7 +29,7 @@ public with sharing class AdyenClient {
 
     private HttpResponse sendRequest(HttpRequest request){
         Http httpClient = new Http();
-        return httpClient.send(request);
+        return (new CommercePayments.PaymentsHttp()).send(request);
```

Comment:
Can we safely replace the `httpClient` for `new CommercePayments.PaymentsHttp()` for all packages? If so, we can also remove the declaration of `httpClient`.

Reply:
Done. Yes, it is safe to replace.